packages/product-feed-heureka/composer.json

```diff
@@ -30,19 +30,19 @@
         "doctrine/dbal": "^2.7",
         "doctrine/doctrine-bundle": "^1.8",
         "shopsys/doctrine-orm": "2.6.2",
-        "shopsys/form-types-bundle": "dev-master",
-        "shopsys/framework": "dev-master",
-        "shopsys/migrations": "dev-master",
-        "shopsys/plugin-interface": "dev-master",
+        "shopsys/form-types-bundle": "7.0.0-beta2",
+        "shopsys/framework": "7.0.0-beta2",
+        "shopsys/migrations": "7.0.0-beta2",
+        "shopsys/plugin-interface": "7.0.0-beta2",
         "symfony/symfony": "^3.4.8"
     },
     "require-dev": {
         "phpunit/phpunit": "^6.0|^7.0",
-        "shopsys/coding-standards": "dev-master",
+        "shopsys/coding-standards": "7.0.0-beta2",
         "twig/extensions": "^1.5.1",
         "twig/twig": "^2.4.8"
     },
     "conflict": {
-        "symfony/dependency-injection": ">=3.4.15"
+        "symfony/dependency-injection": "3.4.15||3.4.16"
```

Comment:
i found some article now about double pipes and used version now is with just single pipe
`"3.4.15||3.4.16"` -> `"3.4.15|3.4.16"` it will be more consistent with `"phpunit/phpunit": "^6.0|^7.0",`